docs/content/integrations/embedded-elt.mdx

```diff
@@ -249,12 +244,12 @@ replication_config = {
 }
 
 
-@sling_assets(replication_config=replication_config)
+@sling_assets(
+    replication_config=replication_config,
+    dagster_sling_translator=DagsterSlingTranslator(),
```

Comment:
reminder to remove these other unnecessary uses of `dagster_sling_translator` before landing